src/main/java/org/broadinstitute/hellbender/utils/dataflow/BucketUtils.java

```diff
@@ -45,8 +44,12 @@ public static boolean isHadoopUrl(String path) {
      */
     public static InputStream openFile(String path, PipelineOptions popts) {
         try {
+            if (null==path) throw new NullPointerException("path was null, shouldn't be.");
+            if (null==popts) throw new NullPointerException("popts was null, shouldn't be.");
             if (BucketUtils.isCloudStorageUrl(path)) {
-                return Channels.newInputStream(new GcsUtil.GcsUtilFactory().create(popts).open(GcsPath.fromUri(path)));
+                GcsPath gcsPath = GcsPath.fromUri(path);
+                if (null==gcsPath) throw new NullPointerException("parsing path yielded null, shouldn't have.");
```

Comment:
Intellij doesn't think this condition is possible, and I don't see how it could be possible either since the code it's calling into is returning the result of `new GcsPath`.  Is there some way we get a null here that I can't see?